config/gulp/javascript.js

```diff
@@ -2,7 +2,6 @@
 const buffer = require("vinyl-buffer");
 const browserify = require("browserify");
 const childProcess = require("child_process");
-const eslint = require("gulp-eslint");
```

Comment:
Removed gulp-eslint as it wasn't working properly and we don't need gulp to run eslint